contracts/exchange/ExchangeAccessFacet.sol

```diff
@@ -5,16 +5,21 @@ import '../lib/LibAddressSet.sol';
 import '../lib/LibPaginate.sol';
 import '../spc/SpcTopFacet.sol';
 import '../interfaces/IHasMembers.sol';
+import '../interfaces/IHasActivations.sol';
 import './lib/LibExchangeAccess.sol';
 import './lib/AExchangeFacet.sol';
 
 
 /** @title The Exchange Smart Contract.
  *  @dev The Exchange Access facet is in charge of keeping track of exchange members.
  */
-contract ExchangeAccessFacet is AExchangeFacet, IHasMembers {
+contract ExchangeAccessFacet is AExchangeFacet, IHasMembers, IHasActivations {
   using LibAddressSet for LibAddressSet.Data;
 
+  // Member activation / deactivation.
+  event MemberActivated(address indexed member);
```

Comment:
Remember - events should be declared in this case in `IFastEvents` and not duplicated here.

Reply:
Yep, I was getting some errors so I'd moved them back... it was probably transient though so I'll put them back in `IExchangeEvents` 👍